master/buildbot/reporters/github.py

```diff
@@ -81,7 +81,8 @@ def createStatus(self,
             [self.baseURL, 'repos', repo_user, repo_name, 'statuses', sha]), json=payload)
 
     @defer.inlineCallbacks
-    def send(self, build):
+    def send(self, postData):
```

Comment:
please don't change the semantic. its a build that is sent in send.

having variable argument mean in a method is not a good thing. We can make two. one for build (sendForBuild), and one for buildset (sendForBuildSet)